### DISCUSSION TITLE
Why can't TLite choose CRSF  in internal RF

### DISCUSSION THREAD

**Discussion by linplusling** [EdgeTX/edgetx#1193] [Q&A] [answered]
I had flash tlite to 2.5,even 2.6,but i still can't choose CRSF  in internal RF,can anybody teach me how to solve this problem

**Comment by raphaelcoeffic** [comment, marked as answer]
> I had flash tlite to 2.5,even 2.6,but i still can't choose CRSF in internal RF,can anybody teach me how to solve this problem

The latest nightly should have it, which version did you check?

**Reply by linplusling** [reply]
i had flash 2.5,2.5rc1/rc2/rc3 and the latest nightly, all of them can't choose CRSF. In the latest the internal RF can't choose anything only one choice that is off.

**Reply by raphaelcoeffic** [reply]
First, it will only work with the latest nightly. Second, the choice you are looking for is in the Hardware Settings. Is it where you are looking for it?

**Reply by pfeerick** [reply]
It is there ;) 

![image](https://user-images.githubusercontent.com/5500713/144008674-051552cd-59c3-4523-a525-98f9f2f8f2c7.png)

**Reply by linplusling** [reply]
thank you ifound it

**Reply by dancingpizza** [reply]
@pfeerick thank you! I waste whole day, and finally found it!!!
